design_linked_list.py

```diff
@@ -0,0 +1,73 @@
+class Node:
+    def __init__(self, val = 0, next = None):
+        self.val = val
+        self.next = next
+        
+class MyLinkedList:
+
+    def __init__(self):
+        self.head = Node()
+        self.length = 0
+
+    def get(self, index: int) -> int:
+        if index >= self.length:
+            return -1
+        
+        cur = self.head.next
+        for _ in range(index):
```

Comment:
having _ as a variable is a better approach when we are not going to use it inside of the code